src/init.cpp

```diff
@@ -94,9 +99,14 @@ template <>
 struct Init<Backend::GPU> {
   static void initialize(const configuration& cfg) {
     const int device = 0;
+    // setup polling on default pool, enable exceptions and init mpi internals
+    pika::mpi::experimental::init(false, true);
+    pika::mpi::experimental::register_polling();
+
```

Comment:
This is not necessary. The CPU backend is always initialized and takes care of registering for polling.